rotkehlchen/chain/l2_with_l1_fees/types.py

```diff
@@ -0,0 +1,71 @@
+from typing import Literal
+
+from rotkehlchen.types import (
+    ChainID,
+    ChecksumEvmAddress,
+    EvmTransaction,
+    EVMTxHash,
+    SupportedBlockchain,
+    Timestamp,
+)
+
+L2_CHAINS_WITH_L1_FEES = [
+    SupportedBlockchain.OPTIMISM,
+    SupportedBlockchain.BASE,
+    SupportedBlockchain.SCROLL,
+]
+
+SupportedL2WithL1FeesChain = Literal[
+    SupportedBlockchain.OPTIMISM,
+    SupportedBlockchain.BASE,
+    SupportedBlockchain.SCROLL,
+]
+
+L2_CHAIN_IDS_WITH_L1_FEES = [ChainID.OPTIMISM, ChainID.BASE, ChainID.SCROLL]
+
+SupportedL2WithL1FeesChainId = Literal[ChainID.OPTIMISM, ChainID.BASE, ChainID.SCROLL]
```

Comment:
This can be simplified. No need to repeat yourself.

```suggestion
L2_CHAINS_WITH_L1_FEES_TYPE = Literal[
    SupportedBlockchain.OPTIMISM,
    SupportedBlockchain.BASE,
    SupportedBlockchain.SCROLL,
]
L2_CHAINS_WITH_L1_FEES: tuple[L2_CHAINS_WITH_L1_FEES_TYPE, ...] = typing.get_args(L2_CHAINS_WITH_L1_FEES_TYPE )  # noqa: E501

L2_CHAINIDS_WITH_L1_FEES_TYPE = Literal[ChainID.OPTIMISM, ChainID.BASE, ChainID.SCROLL]
L2_CHAINIDS_WITH_L1_FEES: tuple[L2_CHAINIDS_WITH_L1_FEES_TYPE, ...] = typing.get_args(L2_CHAINIDS_WITH_L1_FEES_TYPE )  # noqa: E501
```

Reply:
oh, nice! I spend half an hour trying to get ChatGPT to suggest me a solution for removing repetition. gave up in the end 😆